### ISSUE TITLE
Angular material expansion panel inside accordion - dragged item looks like two items being dragged

### ISSUE THREAD

**Issue by Hainesy** [SortableJS/angular-sortablejs#102]
I have an Angular material accordion containing expansion panels. The dragging and sorting all appears to work fine, but the actual image when dragging makes it look like two items are being moved (see attached image).

![image](https://user-images.githubusercontent.com/520883/39179857-502a961a-47ad-11e8-8d5b-3f47d32c64a0.png)

**Comment by smnbbrv** [comment]
Seems like a duplicate of #60

**Comment by smnbbrv** [comment]
Could you confirm? Is there a weird div created inside the parent?

**Comment by Hainesy** [comment]
I don't believe it's the same issue as it correctly sorts the array. It's not a functional issue, purely a visual one.

Here's what gets ouput (I've removed the irrelevant cruft and also replaced the options [object] so you can see them)

```
<mat-accordion multi="false" ng-reflect-sortablejs="sections"
  ng-reflect-input-options="{ handle: '.drag-category', onSort: onSort.bind(this) }">
  <mat-expansion-panel draggable="false">
    <mat-expansion-panel-header>
      <span class="mat-content">
        <mat-panel-titl>
          <mat-icon class="drag-category">drag_handle</mat-icon>
          <span class="category-name">APERITIFS</span>
        </mat-panel-title>
        <mat-panel-description _ngcontent-c40="" class="justify-content-between align-items-center mat-expansion-panel-header-description">
          ...
        </mat-panel-description>
      </span>
    </mat-expansion-panel-header>
    <div class="mat-expansion-panel-content" role="region">
		...
    </div>
  </mat-expansion-panel>
</mat-accordion>
```

**Comment by smnbbrv** [comment]
Hi @Hainesy 

please check https://github.com/SortableJS/angular-sortablejs/blob/master/README.md#angular-material-specifics 

maybe helps to solve your issue. If you find a solution, feel free to share!

**Comment by Hainesy** [comment]
@smnbbrv I'm afraid that didn't help.

In fact since upgrading to v6, the problem seems to have worsened such that more than two items appear to be dragged simultaneously now.

![image](https://user-images.githubusercontent.com/520883/40910898-43a6a764-67e5-11e8-836a-7f43cea64eb9.png)

**Comment by wkndmarkt** [comment]
I've got the same problem.
It may be a problem with webkit since it appears only on Chrome and Safari.

**Comment by oleksbud** [comment]
I've got the same problem.

**Comment by smnbbrv** [comment]
That’s a WebKit bug. If you put a simple dragable property on that element it behaves the same. What you can do is force a fallback from sortablejs properties and it would work fine without the bug because it wouldn’t internally use draggable. However the performance is bad then. I didn’t find a fix for now yet

**Comment by oleksbud** [comment]
@smnbbrv you a right, the native drag&drop API is usefull here.

I've changed the dragImage:

dragStart(e) => {
this.myImage=e.srcElement.getElementsByTagName('mat-expansion-panel-header')[0].cloneNode(true);
document.body.appendChild(this.myImage);
e.dataTransfer.setDragImage(this.dragImage,0,0)
}

dragEnd(e) => {
document.body.removeChild(this.myImage)
}

**Comment by smnbbrv** [comment]
@oleksbud does it work in the end of the day? If yes, could you be so kind to create a stackblitz that I can add it to the readme?

**Comment by oleksbud** [comment]
https://stackblitz.com/edit/angular-bttxvh?file=src%2Fapp%2Fpanels-component%2Fpanels.ts

There are some issues with dependensies in stackblitz, but the main principle is visible.

**Comment by smnbbrv** [comment]
hm, the question is whether the binding is kept because it feels like sortablejs does not come into play here anymore... I would need some time to investigate this further. Thank you for the help @oleksbud !

**Comment by chany-dolg** [comment]
> I don't believe it's the same issue as it correctly sorts the array. It's not a functional issue, purely a visual one.
> 
> Here's what gets ouput (I've removed the irrelevant cruft and also replaced the options [object] so you can see them)
> 
> ```
> <mat-accordion multi="false" ng-reflect-sortablejs="sections"
>   ng-reflect-input-options="{ handle: '.drag-category', onSort: onSort.bind(this) }">
>   <mat-expansion-panel draggable="false">
>     <mat-expansion-panel-header>
>       <span class="mat-content">
>         <mat-panel-titl>
>           <mat-icon class="drag-category">drag_handle</mat-icon>
>           <span class="category-name">APERITIFS</span>
>         </mat-panel-title>
>         <mat-panel-description _ngcontent-c40="" class="justify-content-between align-items-center mat-expansion-panel-header-description">
>           ...
>         </mat-panel-description>
>       </span>
>     </mat-expansion-panel-header>
>     <div class="mat-expansion-panel-content" role="region">
> 		...
>     </div>
>   </mat-expansion-panel>
> </mat-accordion>
> ```
